mm2src/coins/z_coin/storage.rs

```diff
@@ -1,5 +1,204 @@
+use crate::z_coin::ValidateBlocksError;
+use crate::z_coin::ZcoinConsensusParams;
+
 pub mod blockdb;
 pub use blockdb::*;
 
 pub mod walletdb;
 pub use walletdb::*;
+
+#[cfg(target_arch = "wasm32")]
+use walletdb::wasm::storage::DataConnStmtCacheWasm;
+#[cfg(debug_assertions)]
+use zcash_client_backend::data_api::error::Error;
```

Comment:
This will never compile.

https://github.com/KomodoPlatform/komodo-defi-framework/blob/79f620559a635704959d3e88067f7ffe5ee30f1f/Cargo.toml#L51-L70

Reply:
thanks. removed [cfg(debug_assertions)]